test/rubocop/cop/minitest/assert_equal_test.rb

```diff
@@ -133,6 +152,27 @@ def test_do_something
     RUBY
   end
 
+  def test_registers_offense_when_using_assert_operator_with_negated_equal_symbol_argument
+    assert_no_offenses(<<~RUBY)
+      class FooTest < Minitest::Test
+        def test_do_something
+          assert_operator('rubocop-minitest', :!=, actual)
+        end
+      end
+    RUBY
+  end
+
+  # Redundant parentheses should be removed by `Style/RedundantParentheses` cop.
```

Comment:
NOTE: This case will be solved by https://github.com/rubocop/rubocop/pull/12247.